validator/validate.py

```diff
@@ -0,0 +1,83 @@
+"""Configuration files validator"""
+
+import os
+from pathlib import Path
+
+import jsonschema
+import yaml
+
+from validator.config.config import Configuration, SupportedValidations
+from validator.errors.errors import InvalidTargetPathError
+from validator.schemas.schema import SCHEMAS
+
+
+def build_app_path():
```

Comment:
```suggestion
def build_apps_path():
```
```suggestion
def build_apps_root_path():
```
Or the same with `get_` 🤷  ?